worlds/sc2/items.py

```diff
@@ -1902,6 +1905,59 @@ def get_full_item_list():
         ItemData(812 + SC2LOTV_ITEM_ID_OFFSET, ProtossItemType.Solarite_Core, 12, SC2Race.PROTOSS, origin={"ext"}),
 }
 
+# Add keys to item table
+# Mission keys (1-999)
+mission_key_item_table = {
+    item_names._TEMPLATE_MISSION_KEY.format(mission.mission_name):
+        ItemData(mission.id + SC2_KEY_ITEM_ID_OFFSET, FactionlessItemType.Nothing, 0, SC2Race.ANY,
+                 classification=ItemClassification.progression, quantity=0, origin={"key"})
```

Comment:
`origin` is unused and probably better not to add more. We should get around to removing it entirely, but as that would modify most lines of this file it's probably better to wait until a time where no items are being added.